### PR TITLE
Update `CompatHelper` workflow

### PR DESCRIPTION
**Pull request by prbzrg**
I added other directories to check by `CompatHelper`. Also, I enabled the check for jlls.
Before merging this, you should know that at first, it will create many PRs.

### CODE REVIEW

File: .github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -8,19 +8,28 @@ jobs:
...
+      - name: Pkg.add
+        shell: julia --color=yes {0}
+        run: |
+          import Pkg
+          Pkg.add("CompatHelper")
+      - name: CompatHelper.main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+          COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
```
Comment (vchuravy) [inline .github/workflows/CompatHelper.yml:31]: Why this change?

Comment (vchuravy) [inline .github/workflows/CompatHelper.yml:31]:
```suggestion
```

Comment (prbzrg) [reply .github/workflows/CompatHelper.yml]: If I remember correctly, without it, there was an error about permissions. Let's delete it for now.

Comment (prbzrg) [reply .github/workflows/CompatHelper.yml]: Albeit, `CompatHelper` has it
https://github.com/JuliaRegistries/CompatHelper.jl/blob/e72bd6b815a69d14ccdb0c85b3589bed2a2af704/.github/workflows/CompatHelper.yml#L44

**Review by vchuravy — APPROVED** [review]

**Comment by wsmoses** [comment]
@vchuravy @prbzrg can this not open PR's for test which have compat for the main repo?

We shouldn't keep separate compats for both

**Comment by vchuravy** [comment]
Yeah I agree that is suboptimal.

**Comment by vchuravy** [comment]
There is an upstream issue for that: https://github.com/JuliaRegistries/CompatHelper.jl/issues/294

**Comment by vchuravy** [comment]
I disabled it for `test` again
